Gruntfile.coffee

```diff
@@ -41,6 +48,12 @@ module.exports = (grunt) ->
     shell:
       webdriverManagerUpdate:
         command: "node_modules/grunt-protractor-runner/node_modules/.bin/webdriver-manager update"
+      generateApiDocs:
+        command: [
+          "node_modules/docha/bin/docha -p '.tmp/elementHelper.spec.js' -o 'docs/elementHelper.md' -e _"
```

Comment:
nit: It might be helpful to have a general description of each of these and a link to them in the main readme or a main entry point readme for the documentation.